fhir/src/services/oauth/oauth.validator.js

```diff
@@ -0,0 +1,37 @@
+const clients = require('./client.service');
+
+/**
+ * @name getClient
+ * @description Gets the necessary client information so the auth service can
+ * validate the token and signature.
+ * In our case, we can look up the client information directly for this stub.
+ *
+ * @param {Winston} logger - Winston logger
+ * @param {Object} config - FHIR Core server config object
+ * @param {Object} options - Necessary options for generating the code
+ * @return {Promise}
+ */
+module.exports.getClient = (clientId) => new Promise((resolve, reject) => {
+	clients.findByClientId(clientId).then((client) => {
```

Comment:
`clients.findByClientId` already returns a promise, right?  Just return it, don't wrap it in another promise